articles/chavez2.xml

```diff
@@ -33,23 +33,31 @@
       
       <body>
          <div type="article"><!-- Enclose each paragraph of your text in a new <p></p> tag. -->
-            <p>[Article text goes here...]</p>
+            <p>The artifact below is a letter written by Cesar Chavez. To better understand it, we need to learn a few things about Chavez and his monumental unionization of farm workers.  
```

Comment:
each paragraph should be enclosed in its own set of `<p></p>` tags - you have everything in one tag with extra spaces and returns. your page currently isn't loading and this could be why